markdown/metadata.go

```diff
@@ -8,8 +8,13 @@ import (
 
 // Metadata is document metadata in the "front matter" of a Markdown document.
 type Metadata struct {
-	Title                       string `yaml:"title"`
-	IgnoreDisconnectedPageCheck bool   `yaml:"ignoreDisconnectedPageCheck"`
+	IgnoreDisconnectedPageCheck bool `yaml:"ignoreDisconnectedPageCheck"`
+
+	Title       string   `yaml:"title"`
+	Description string   `yaml:"description"`
+	Category    string   `yaml:"category"`
+	Type        string   `yaml:"type"`
+	Tags        []string `yaml:"tags"`
```

Comment:
@jhchabran Could we add an image field as well.

```
Image string `yaml:"image"`
```

Reply:
Ack, I'll aslo add the canonical path along the way 👍  !